lib/open_api_spex/cast/object.ex

```diff
@@ -26,6 +26,7 @@ defmodule OpenApiSpex.Cast.Object do
     end
   end
 
+  defp check_unrecognized_properties(%{schema: %{additionalProperties: true}} = ctx, _), do: :ok
```

Comment:
This is now in master: Fixed here https://github.com/open-api-spex/open_api_spex/pull/148/files#diff-90b814733e17cc4503841611aa02c4fbR30

Reply:
awesome, thanks!